File: tree.js
const line =  5;
for (let i = 0; i<line; i++) {
    for(let j=line-i-1; j>0; j--) {
        process.stdout.write(" ");
    }
    for(let star=0; star<2*i+1; star++) {
        process.stdout.write("*");
    }
    console.log();
    if (i == line-1) {
        for(let j=line-2; j>0; j--) {
            process.stdout.write(" ");
        }
        console.log("|_|");
    } 
}

File: tree.py
line = 5
for i in range(0,line):
    for j in range(line-1-i, 0, -1):
        print(" ", end="")
    for star in range(0, 2*i+1):
        print("*", end="")
    print("")
    if i == line-1:
        for j in range(line-2, 0, -1):
            print(" ", end="")    
        print("|_|")

